packages/xo-web/src/xo-app/index.js

```diff
@@ -272,6 +284,23 @@ export default class XoApp extends Component {
                   </button>
                 </div>
               )}
+              {this.isTrialVersion() && !this.state.dismissedTrialBanner && (
+                <div className='alert alert-info mb-0'>
+                  <a
+                    href='https://xen-orchestra.com/#!/xoa?pk_campaign=xo_source_banner'
+                    rel='noopener noreferrer'
+                    target='_blank'
+                  >
+                    {_('isTrialLicense', {
+                      edition: trial.trial.productId,
```

Comment:
Use `xoa-plans` to find the actual name of the product instead of the ID.

Reply:
If am using ‘getXoaPlan()‘ from ‘xoa-plans‘, this return me the current edition and not the trial edition. Is that what we want to show in the banner ? ( eg: You are currently in a Free trial period that will end on Nov 26, 2020 )

Reply:
You can call `getXoaPlan` with the `plan` you're interested in, which you can get with `productId2Plan`. So something like `getXoaPlan(productId2Plan[productId])`.